vignettes/syntax.Rmd

```diff
@@ -0,0 +1,398 @@
+---
+title: "The tidyselect syntax"
```

Comment:
I think you need a name more like "tidyselect details" to make it clear that this is aimed at the package author, not the the data scientist.

Reply:
This is now `Technical description of tidyselect`